testsuite/features/build_validation/init_clients/slemicro52_ssh_minion.feature

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2023 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+# Beware: After altering the system e.g. package installation/removal, the system
+#         has to be rebooted to take effect due to the nature of SLE Micro.
+
+@slemicro52_ssh_minion
+Feature: Bootstrap a SLE Micro 5.2 Salt SSH minion
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Bootstrap a SLE Micro 5.2 SSH minion
+    When I follow the left menu "Systems > Bootstrapping"
+    Then I should see a "Bootstrap Minions" text
+    When I check "manageWithSSH"
+    And I enter the hostname of "slemicro52_ssh_minion" as "hostname"
+    And I enter "linux" as "password"
+    And I select "1-slemicro52_ssh_minion_key" from "activationKeys"
+    And I select the hostname of "proxy" from "proxies" if present
+    And I click on "Bootstrap"
+    And I wait until I see "Successfully bootstrapped host!" text
+    And I wait until onboarding is completed for "slemicro52_ssh_minion"
+
+  Scenario: Reboot the SLE Micro 5.2 SSH minion and wait until reboot is completed
```

Comment:
When SSH Minion, we don't need:
`# 2 reboots are necessary after bootstrapping  ?`

Reply:
Good point. I did not tetst that. Can you confirm it is not necessary or is it just an assumption?

Reply:
I assume it is 👀

Reply:
Apparently this is not needed. See https://suse.slack.com/archives/C02D12TNYLS/p1674826501515899